.storybook/recipes/NumberIconList/NumberIconList.tsx

```diff
@@ -0,0 +1,26 @@
+import clsx from 'clsx';
+import React, { ReactNode } from 'react';
+import styles from './NumberIconList.module.css';
+
+export interface Props {
+  /**
+   * Slot for adding number icon elements
+   */
+  children?: ReactNode;
```

Comment:
Is children supposed to be optional? If so what does it look like if children is `undefined`?